atoma-state/src/state_manager.rs

```diff
@@ -1401,25 +1410,29 @@ impl AtomaState {
     pub async fn insert_new_stack(&self, stack: Stack) -> Result<()> {
         sqlx::query(
             "INSERT INTO stacks
-                (owner_address, stack_small_id, stack_id, task_small_id, selected_node_id, num_compute_units, price_per_one_million_compute_units, already_computed_units, in_settle_period, total_hash, num_total_messages)
-            VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)
+                (owner_address, stack_small_id, stack_id, task_small_id, selected_node_id, 
+                num_compute_units, price_per_one_million_compute_units, already_computed_units, 
+                in_settle_period, total_hash, num_total_messages, is_confidential)
+            SELECT 
```

Comment:
Is this a valid query? `insert into <table> select ...` ?

Reply:
Look at the unit tests